cfstore/plugins/ssh.py

```diff
@@ -187,9 +200,26 @@ def run_script(self, remotepath, collection, script):
             print("err:",line)
         for line in stdout:
             print("out:",line)
-        with open("sample.json","w") as writepath:
-            json.dump(stdout,writepath) 
-        self._sftp.remove(remotescript)
+        self._sftp.remove(remotepath+scriptname)
+
+    def aggregateFiles(self, remotepath):
```

Comment:
Where did we leave the conversation about what if the remote aggregation file already exists?